Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo()
+        {
+            const int maxEntries = 2;
+
+            for (int i = 1; i <= maxEntries; i++)
+            {
+                Console.Clear();
+
+                RegisterNewPersonData(i);
+
+                PromptNewPerson(i);
+            }
+        }
+
+        private static string WriteNewLine(int count)
+        {
+            StringBuilder newLines = new StringBuilder(Environment.NewLine);
+
+            for (int i = 1; i < count; i++)
+            {
+                newLines.Append(Environment.NewLine);
+            }
+
+            return newLines.ToString();
+        }
+
+        static void RegisterNewPersonData(int entry)
+        {
+            Console.WriteLine($"Enter the following information for person #{entry}.");
+
```

Comment:
This is a good example of separating stuff with an empty line. You actually separated concepts.